modelci/hub/converter.py

```diff
@@ -123,6 +122,20 @@ def from_onnx(
 
 __all__ = ['TorchScriptConverter', 'TFSConverter', 'ONNXConverter', 'TRTConverter', 'to_tvm']
 
+class PYTConverter(object):
```

Comment:
it should be named as `PyTorchExporter` and put somewhere else.

Reply:
> it should be named as `PyTorchExporter` and put somewhere else.
How about just put it into register process?

Reply:
good